Sources/Tools/MaxComponent/CMakeLists.txt

```diff
@@ -107,7 +106,6 @@ set(MaxComponent_SOURCES
     plBehavioralComponents.cpp
     plBipedKiller.cpp
     plBlowComponent.cpp
-    plCameraComponent.cpp
```

Comment:
Not sure how the deletion of plCameraComponent is related to PCH/Unity builds...  Perhaps this should be a separate PR?

Reply:
This file's contents were a duplicated subset of plCameraComponents.cpp, causing the unity build to fail. This was previously a warning per #689.

Reply:
*sigh*